.travis.yml

```diff
@@ -24,5 +24,7 @@ env:
 # Required for lock file generation
 # https://github.com/greenkeeperio/greenkeeper-lockfile
 before_install: yarn global add greenkeeper-lockfile@1
-before_script: greenkeeper-lockfile-update
+before_script: 
+  - greenkeeper-lockfile-update
+  - yarn storybook & sleep 7
```

Comment:
`sleep 7`? any way to make this a bit more deterministic?

Reply:
It is a bit difficult as you don't know how long will it take to be built. I found this workaround when I used cypress 

I can try another approach `-- --silent &`

Reply:
Previously we've used something to keep trying to ping the server until it responds

Reply:
Do you still have the command?

Reply:
hmm - even in Travis documentation - they still use sleep:
https://docs.travis-ci.com/user/gui-and-headless-browsers/